src/linear_algebra/general_linear_group.lean

```diff
@@ -0,0 +1,200 @@
+/-
+Copyright (c) 2021 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+
+The General Linear group $GL(n, R)$
```

Comment:
```suggestion
```
We don't put comments here (any more), it's only the comment below that ends up in doc-gen

Reply:
done